client/signup/steps/user/index.jsx

```diff
@@ -36,6 +36,51 @@ import { getSuggestedUsername } from 'calypso/state/signup/optional-dependencies
 import { saveSignupStep, submitSignupStep } from 'calypso/state/signup/progress/actions';
 import './style.scss';
 
+function getLoginLink( props ) {
+	// TODO: If Reskin signup experiment wins, then refactor to remove duplicate definition of this function
+	// in <SignupForm> component
+	return login( {
+		isJetpack: 'jetpack-connect' === props.sectionName,
+		from: props.from,
+		redirectTo: getRedirectToAfterLoginUrl( props ),
+		locale: props.locale,
+		oauth2ClientId: props.oauth2Client?.id,
+		wccomFrom: props.wccomFrom,
+	} );
+}
+
+function getRedirectToAfterLoginUrl( props ) {
```

Comment:
Since we're extracting this function out of the component, maybe we can consider using explicit args rather than `props` object. This is just a suggestion so it's not blocking.

```suggestion
function getRedirectToAfterLoginUrl( oauth2Signup, query, flowName, stepName, userLoggedIn  ) {
```

Reply:
Yeah, not a bad idea. I've been thinking about destructuring the props here instead - so it's easier to also type down the line. 

So `getRedirectToAfterLoginUrl( { oauth2Signup, query, flowName, stepName, userLoggedIn }  ) ` instead?